components/FlexibleHeader/src/MDCFlexibleHeaderViewController.m

```diff
@@ -207,19 +199,94 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView
   }
 }
 
+#pragma mark - Top layout guide support
+
+// Whenever the flexible header's height changes, we want to adjust the topLayoutGuide length
```

Comment:
Done.

Reply:
Historically we've only used // comments in implementation files because `/** */` blocks otherwise get picked up in documentation generators. I've updated this to use a `/* */` block instead, but there isn't a particularly strong style requirement here.

Reply:
Ah! Np I misread that as well.